crates/uv-cli/src/lib.rs

```diff
@@ -4324,6 +4324,27 @@ pub struct PublishArgs {
     )]
     pub token: Option<String>,
 
+    /// Always use trusted publishing through GitHub Actions.
+    ///
+    /// By default, uv checks for trusted publishing when running in GitHub Actions, but ignores it
```

Comment:
Self-note: this changes the default flow of the publishing logic without having the user to opt-in into it, and I was wondering whether this a security surprise/hazard.
I convinced myself that this should be fine because, although this starts automatically using ambient capabilities that are provided (by default?) by GH and that the users may not be fully aware of, completing a proper authentication loop requires a setup with an explicit opt-in step by the user on the [PyPi side](https://docs.pypi.org/trusted-publishers/adding-a-publisher/).